pkg/sqlite/load.go

```diff
@@ -61,7 +61,7 @@ func (s *SQLLoader) AddOperatorBundle(bundle *registry.Bundle) error {
 }
 
 func (s *SQLLoader) addOperatorBundle(tx *sql.Tx, bundle *registry.Bundle) error {
-	addBundle, err := tx.Prepare("insert into operatorbundle(name, csv, bundle, bundlepath, version, skiprange, replaces, skips) values(?, ?, ?, ?, ?, ?, ?, ?)")
+	addBundle, err := tx.Prepare("insert into operatorbundle(name, csv, bundle, bundlepath, version, skiprange, replaces, skips, dependencies) values(?, ?, ?, ?, ?, ?, ?, ?, ?)")
```

Comment:
Is this the right place to put this? I would like to see how the querier interface works here as well before we add it as just a serialized blob in a field on the bundle. Doesn't it make more sense to denormalize this somewhat and create a separate table on the database so that you can write queries explicitly about a set or subset of dependency information?